step-templates/mariadb-create-database.json

```diff
@@ -0,0 +1,73 @@
+{
+    "Id": "2bdfe600-e205-43f9-b174-67ee5d36bf5b",
+    "Name": "MariaDB - Create Database If Not Exists",
+    "Description": "Creates a MariaDB database if it doesn't already exist.  This template is also compatible with MariaDB.",
```

Comment:
I'm not familiar with MariaDB but this description doesn't make sense to me (or James!)

Reply:
Copy/paste PICNIC error.